### PR TITLE
FAQ: Add some more variants on how to deal with temporary files

### PR DESCRIPTION
**Pull request by PhilipMetzger**
The people in #323 think that we should document all patterns there are to deal with ignored files.

# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -104,7 +104,9 @@ files you want to keep around to match the pattern.
 
 If `$EDITOR` integration is important, something like `scratchpad.*` may be more
 helpful, as you can keep the filename extension intact (it
-matches `scratchpad.md`, `scratchpad.rs` and more).
+matches `scratchpad.md`, `scratchpad.rs` and more). Another option is to add a 
+directory to the global `.gitignore` which then stores all your temporary files
+and notes. To achieve this add `<your-directory>/` to the`.git/info/exclude`.
```
Comment (steveklabnik) [inline docs/FAQ.md:109]: I would write this like this

```suggestion
and notes. To achieve this add `<your-directory>/` to `.git/info/exclude`.
```

Comment (steveklabnik) [inline docs/FAQ.md:109]: If you prefer the 'the', there's also a space needed here, so you could choose this version too.

```suggestion
and notes. To achieve this add `<your-directory>/` to the `.git/info/exclude`.
```

Comment (PhilipMetzger) [reply docs/FAQ.md:109]: Thanks Steve, I choose your first suggestion.

File: docs/FAQ.md [hunk 2]
```diff
@@ -104,7 +104,9 @@ files you want to keep around to match the pattern.
 
 If `$EDITOR` integration is important, something like `scratchpad.*` may be more
 helpful, as you can keep the filename extension intact (it
-matches `scratchpad.md`, `scratchpad.rs` and more).
+matches `scratchpad.md`, `scratchpad.rs` and more). Another option is to add a 
+directory to the global `.gitignore` which then stores all your temporary files
+and notes. To achieve this add `<your-directory>/` to `.git/info/exclude`.
```
Comment (ilyagr) [inline docs/FAQ.md:109]: The global gitignore is `~/.git/ignore`, I think. 

(Optional) You could also link https://git-scm.com/docs/gitignore there. Finally, I would be slightly more concrete:

> For example, you could add `scratch/` to [`~/.git/ignore`](https://git-scm.com/docs/gitignore) and then store arbitrary files in `your-git-repo/scratch/`.

**Review by ilyagr — APPROVED** [review]
Thank you!

**Pull request merged**